nbd/nbd-package/DEBIAN/postinst

```diff
@@ -6,4 +6,12 @@ chmod +x /usr/bin/map_curve_disk.sh
 cp /usr/bin/map_curve_disk.sh /etc/init.d
 update-rc.d map_curve_disk.sh defaults
 
+CURVETAB=/etc/curve/curvetab
+CURVETABBAK=/etc/curve/curvetab-bak
+
+if [ -f ${CURVETABBAK} ]; then
+    cp -f ${CURVETABBAK} ${CURVETAB}
```

Comment:
Add '-p' cp command option to preserve 'mode,ownership,timestamps' ?

Reply:
fix